string object_name

---

int64 code
string message
temoto_core/Error[] error_stack
